src/components/datepicker/datepicker.js

```diff
@@ -564,6 +566,18 @@ DatePicker.prototype = {
     }
   },
 
+  /**
+   * Set extra attributes.
+   * @private
+   * @returns {void}
+   */
+  setExtraAttributes() {
+    if (!this.settings.attributes) return;
+
+    utils.addAttributes(this.element, this, this.settings.attributes);
+    utils.addAttributes(this.trigger, this, this.settings.attributes);
```

Comment:
@ericangeles I think this might need adjusting.  The automated IDs for each piece of the component need to be different, so that a testing tool targeting elements using this custom ID doesn't accidentally grab the wrong element. The `utils.addAttributes` takes a [fourth argument](https://github.com/infor-design/enterprise/blob/main/src/utils/utils.js#L1307) that acts as a suffix, which can be used to differentiate the two elements.  I'd suggest just adding a `trigger` suffix to the second one:

```suggestion
    utils.addAttributes(this.trigger, this, this.settings.attributes, 'trigger');
```